platform-core-module/src/main/java/com/softicar/platform/core/module/program/execution/ProgramExecutionRunnable.java

```diff
@@ -70,17 +82,20 @@ private AGProgramExecution updateStartedAt() {
 
 	private void executeProgram() {
 
-		try (LogOutputScope scope = new LogOutputScope(logBuffer)) {
+		try (var scope = new LogOutputScope(logOutput)) {
+			Log.finfo("Program started.");
```

Comment:
should we put this text into brackets like the timing as a convention for artificial output?
```
[PROGRAM START]
...
[PROGRAM END]
```

Reply:
ok